python/make_finemap_inputs.py

```diff
@@ -205,6 +200,16 @@ def generate_bed(sumstats,
     if len(bed_frames) > 0:
         bed = BedTool.from_dataframe(pd.concat(bed_frames).sort_values(['chrom', 'start']))
         lead_snps = pd.concat(lead_snps, axis=1).T
+
+        # exclude regions overlapping with the MHC region
+        if exclude_MHC:
+            MHC_bed = BedTool.from_dataframe(pd.DataFrame([6, int(MHC_start), int(MHC_end)], columns=['chrom', 'start', 'end']))
+            overlapping_regions = bed.intersect(MHC_bed, wa=True)
+            if (len(overlapping_regions) > 0):
+                print(overlapping_regions)
+                logger.warning('{} regions excluded due to --exclude-MHC'.format(len(overlapping_regions)))
```

Comment:
could you print the removed regions also here

Reply:
L207 `print(overlapping_regions)`. `overlapping_regions` are the removed regions that are computed via `bed.intersect`. If any, we subtract via  L211`bed.subtract`

Reply:
argh I'm blind the print is already there...